### PR TITLE
Make logs more articulate.

### PR DESCRIPTION
**Pull request by pratikjagrut**
### Motivation

Making logs more articulate.

### Testing

`make test-e2e`

### CODE REVIEW

File: test/e2e/servicebindingrequest_test.go [hunk 1]
```diff
@@ -556,13 +556,15 @@ func serviceBindingRequestTest(
 	}, 120*time.Second, 2*time.Second)
 
 	// after deletion, deployment should not contain envFrom directive anymore
+	t.Log("Deployment should not contain envFrom directive")
 	require.Eventually(t, func() bool {
 		t.Logf("Inspecting deployment '%s'", deploymentNamespacedName)
 		_, err := assertDeploymentEnvFrom(todoCtx, f, deploymentNamespacedName, sbrName)
 		if err != nil {
-			t.Log("expect deployment not to be carrying envFrom directive")
+			t.Log("Expect deployment not to be carrying envFrom directive")
```
Comment (Avni-Sharma) [inline test/e2e/servicebindingrequest_test.go:564]: Can we use t.Logf and print the err as well

**Comment by pratikjagrut** [comment]
4.5-e2e failed due to cluster error:
```
level=error
level=error msg="Error: Unable to find matching route for Route Table (rtb-058197db27d10fcaf) and destination CIDR block (0.0.0.0/0)."
level=error
level=error msg="  on ../tmp/openshift-install-225232763/vpc/vpc-private.tf line 14, in resource \"aws_route\" \"to_nat_gw\":"
level=error msg="  14: resource \"aws_route\" \"to_nat_gw\" {"
level=error
level=error
```
/test 4.5-e2e

**Comment by pratikjagrut** [comment]
e2e failed due to race condition:
```
--- FAIL: TestAddSchemesToFramework (480.18s)
    --- FAIL: TestAddSchemesToFramework/end-to-end (458.28s)
        --- PASS: TestAddSchemesToFramework/end-to-end/scenario-etcd-unannotated-app-db-sbr (32.91s)
        --- PASS: TestAddSchemesToFramework/end-to-end/scenario-db-app-sbr (29.03s)
        --- PASS: TestAddSchemesToFramework/end-to-end/scenario-app-db-sbr (29.02s)
        --- FAIL: TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app (186.70s)
        --- PASS: TestAddSchemesToFramework/end-to-end/scenario-app-sbr-db (41.02s)
        --- PASS: TestAddSchemesToFramework/end-to-end/scenario-sbr-db-app (32.02s)
        --- PASS: TestAddSchemesToFramework/end-to-end/scenario-sbr-app-db (41.00s)
        --- PASS: TestAddSchemesToFramework/end-to-end/scenario-csv-db-app-sbr (32.25s)
        --- PASS: TestAddSchemesToFramework/end-to-end/scenario-csv-app-db-sbr (34.33s)
FAIL
```
```
TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: servicebindingrequest_test.go:333: Waiting for application deployment reach one replica...
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: wait_util.go:67: Waiting for full availability of e2e-application-1288165632807057801 deployment (0/1)
    TestAddSchemesToFramework/end-to-end/scenario-db-sbr-app: servicebindingrequest_test.go:334: 
        	Error Trace:	servicebindingrequest_test.go:334
        	            				servicebindingrequest_test.go:454
        	            				servicebindingrequest_test.go:155
        	            				servicebindingrequest_test.go:97
        	Error:      	Received unexpected error:
```
/test e2e

File: test/e2e/servicebindingrequest_test.go (hunk 1 above)
Comment (pratikjagrut) [reply test/e2e/servicebindingrequest_test.go:564]: Here we are expecting an error, if we get an error the loop will return true so printing an error msg won't be of any use.

**Comment by pratikjagrut** [comment]
Made squash commit.

**Pull request merged**